spreadsheets/sort_rank/landuse.csv

```diff
@@ -21,6 +21,7 @@ dam,*,223
 land,*,222
 wetland,*,220
 mud,*,219
+weir,*,218
```

Comment:
I would put this between `breakwater` and `dam`.

Reply:
I have put `weir` between `breakwater` and `dam`.